user_docs/en/changes.md

```diff
@@ -15,6 +15,11 @@ Unicode CLDR has been updated.
 
 ### New Features
 
+
+* New key commands:
+  * Added unassigned commands for vertical and horizontal scrolling of the mouse wheel to enhance navigation on web pages and apps with dynamic content, such as Dism++. Most mice have a scroll wheel click; by pressing down on the scroll wheel, you can scroll horizontally and vertically simultaneously by moving the cursor in the desired direction. (#16462, @Cary-Rowen)

```

Comment:
I think the information about how mice work is not relevant enough for the changes here. If it's necessary to understand the feature it should be in the user guide.

```suggestion
  * Added unassigned commands for vertical and horizontal scrolling of the mouse wheel, to enhance navigation on web pages and apps with dynamic content, such as Dism++. (#16462, @Cary-Rowen)
```